packages/fast-usdc/src/exos/status-manager.js

```diff
@@ -109,8 +132,34 @@ export const prepareStatusManager = zone => {
        * Add a new transaction with ADVANCED status
        * @param {CctpTxEvidence} evidence
        */
-      advance(evidence) {
-        recordPendingTx(evidence, PendingTxStatus.Advanced);
+      advancing(evidence) {
+        recordPendingTx(evidence, PendingTxStatus.Advancing);
+      },
+
+      /**
+       * @param {NobleAddress} sender
+       * @param {import('@agoric/ertp').NatValue} amount
+       * @param {boolean} success
+       * @throws {Error} if nothing to advance
+       */
+      advanceOutcome(sender, amount, success) {
```

Comment:
it makes sense that this takes a boolean but for the calling code consider having a more explicit `sendAdvanceSuccess` and `sendAdvanceFailure` (which could call this)

Reply:
I'm struggling with this "sendX" phrasing. We're telling the status manager to send something?

Reply:
How about `settleX`? To signify the transfer vow settlement.

I don't mind the Boolean. Another option is a `Fulfilled | Rejected` const that would be more explicit at the call site. 

Also keep in mind we'll soon need `settleForward` (in addition to `settleAdvance` or whatever is decided here)

Reply:
"send" phrasing is from the metaphor of sending a message to the state machine. It's in a bunch of state machine libraries. 

my comment was just a "consider" which is done.